src/components/Facility/FacilityCard.tsx

```diff
@@ -222,38 +223,49 @@ export const FacilityCard = (props: {
                   </div>
                   <div className="flex flex-wrap gap-2">
                     {["DistrictAdmin", "StateAdmin"].includes(userType) && (
-                      <ButtonV2
+                      <Button
                         id="facility-notify"
-                        ghost
-                        border
-                        className="h-[38px]"
+                        variant={"outline_primary"}
+                        className="h-[38px] tooltip"
                         onClick={(_) => setNotifyModalFor(facility.id)}
+                        aria-label={t("notify")}
+                        role="button"
                       >
                         <CareIcon icon="l-megaphone" className="text-lg" />
-                        <span className="hidden md:block">Notify</span>
-                      </ButtonV2>
+                        <span className="hidden sm:inline">{t("notify")}</span>
+                        <span className="tooltip-text tooltip-top sm:hidden">
+                          {t("notify")}
+                        </span>
+                      </Button>
                     )}
-                    <ButtonV2
-                      href={`/facility/${facility.id}`}
+                    <Button
+                      onClick={() => navigate(`/facility/${facility.id}`)}
                       id="facility-details"
```

Comment:
This is not the right way to use buttons as links. Wrap `<Link>` with the button as child. Refer: https://ui.shadcn.com/docs/components/button

<img width="823" alt="image" src="https://github.com/user-attachments/assets/89618ef4-cbfc-46fb-91ba-170025147016" />